server/views/pages/explore.njk

```diff
@@ -28,6 +28,22 @@
     </div>
   </div>
 
+  <div class="row {{ {s:2, m:2, l:2} | spacingClasses({padding:['top']}) }}">
+    <div class="container container--scroll-xl">
+      <div class="grid grid--no-gutters">
+        <div class="{{ {s: 4, m: 8, l: 12, xl: 12} | gridClasses }}">
+          {% set asyncContentEndpoint = '/series-transporter/' + latestDigitalStory %}
+          {% component 'async', { endpoint: asyncContentEndpoint, component: 'series-transporter' } %}
```

Comment:
Ah - [we have a same named thing here](https://github.com/wellcometrust/wellcomecollection.org/pull/911/files#diff-f34dd8a341cd717c49b4bf003a14951bR1) - what are we using that for?